src/pages/Account/AccountSecurity.tsx

```diff
@@ -1,11 +1,70 @@
 import * as React from 'react';
+import { ListGroup, ListGroupItem, Alert } from 'reactstrap';
+import { resetPassword } from 'utils/firebase';
+import { getDisplaySuccessMessage, SuccessMessage, getDisplayErrorMessage, ErrorMessage } from 'utils/validators';
+import { FirebaseConsumer, FirebaseUserProps } from 'HOCs/FirebaseProvider';
+import { User } from 'firebase';
 
-class AccountSecurity extends React.Component<any> {
-  render() {
-    return (
-      <h1>This is the Security Page</h1>
-    );
+interface AlertProperties {
+  color: string;
+  msg: string;
+}
+
+function AccountSecurity() {
+  const [alert, setAlert] = React.useState<AlertProperties>({ color: '', msg: '' });
+  const [isSubmitting, setSubmitting] = React.useState<boolean>(false);
+
+  function handleClick(user: User) {
```

Comment:
i think we can move the handlers below the return now that you're not using `const`

Reply:
👍 moved